### DISCUSSION TITLE
Version 0.11

### DISCUSSION THREAD

**Discussion by misyltoad** [Joshua-Ashton/VPhysics-Jolt#101] [Announcements]
A new release, with a bunch of bug and crash fixes + new features.

## Bug fixes and Improvements
- Fixed prop, player and vehicle damage.
- Fixed a crash/corruption on collision events (#51).
- Fixed triangle winding of CreateDebugMesh triangles (#44).
- Fixed an assertion in GetObjectList.
- Fixed Stop Motion Helper, Ragdoll Mover and Stand Pose Tool (#59 #63).
- Fixed a crash/corruption when constraints are deleted (#17).
- Added support for legacy style .phy files -- fixing crashes when spawn some props/loading some maps (#35).
- Added support for per-wheel brake torque.
- Added a stub for Get/SetPerformanceSettings.
- Added support for SDK 2013 SP and Alien Swarm.
- Added support for Linux on all targeted games/SDKs.
- Added support for older CPUs by adding separate builds for each feature set that is automatically picked (SSE2, SSE4.1 and AVX2)

## Installation

Copy contents of the zip into the game's root directory and replace all files.

<hr /><em>This discussion was created from the release <a href='https://github.com/Joshua-Ashton/VPhysics-Jolt/releases/tag/0.11'>Version 0.11</a>.</em>

**Comment by W1nKille3r** [comment]
damn, that was worth 18 years of waiting

**Comment by splofie** [comment]
hey, not sure if you will see this but will there be a release for mac/do any of these releases work on mac?

**Reply by misyltoad** [reply]
No support for developer-hostile meme platforms is provided.

**Comment by niceEli** [comment]
pls allow older engines i really want to play gmod 9 for args with this

**Comment by Physchh** [comment]
Wanted to ask, if you could somehow get simphys and other vehicle mods to be compatible with this new rewrite? Right now the vehicles fly at moch 10 speed if you try to drive it in any direction with this installed, may be gravity or acceleration related stuff with the physics rewrite?, i havent seen what it's done to any other mods with vehicles, like the LFS planes mod, but if you could really try to get it to be compatible with those mods it'd be pretty wicked dude

**Comment by rrR1238** [comment]
please make a pirated version

**Reply by AlCatSplat** [reply]
How about you buy the game?

**Comment by atomizerofficial** [comment]
Pls fix ragdolls.They like spaghetti or flag physics

**Comment by leuamS** [comment]
Will there be a new version of VPhysics-Jolt anytime soon?

**Comment by rrR1238** [comment]
please fix the loadings I have the game crashes when I re-enter another map

**Comment by leuamS** [comment]
This version is one year old already, I'm losing a bit of hope for a new build

**Reply by YUCLing** [reply]
There are newer builds for Garry's Mod available at https://git.froggi.es/joshua/vphysics_jolt_gmod_builds. For other games and the Source engine, you have to build it yourself.

**Reply by TheCelestial2000** [reply]
> There are newer builds for Garry's Mod available at https://git.froggi.es/joshua/vphysics_jolt_gmod_builds. For other games and the Source engine, you have to build it yourself.


What this new version adds ?
Does fix some bugs ?

**Reply by leuamS** [reply]
While I didn't notice too much, mods like SCars and simfphys work much better

**Reply by YUCLing** [reply]
> > There are newer builds for Garry's Mod available at https://git.froggi.es/joshua/vphysics_jolt_gmod_builds. For other games and the Source engine, you have to build it yourself.
> 
> What this new version adds ? Does fix some bugs ?

As for `50cef3` build, it mainly fixed the collision issue. You can check the full changelog [here](https://github.com/Joshua-Ashton/VPhysics-Jolt/compare/6c3e224...50cef39).
